experiments/compositions/samples/FirstComposition/composition/teampage.yaml

```diff
@@ -16,6 +16,7 @@ apiVersion: composition.google.com/v1alpha1
 kind: Composition
 metadata:
  name: team-page
+ namespace: default
```

Comment:
Wondering if we should start with the CC structure in which case this would be "config-control".

Reply:
Can do. But this should work fine in any namespace. Compositions can exist in any namespace that the admin chooses it to be. The Facades can be in any namespace as well.